src/einsteinpy/tests/test_plotting/test_scattergeodesicplotter.py

```diff
@@ -8,7 +8,10 @@
 
 
 @pytest.fixture()
-def dummy_data():
+def create_test_datas():
```

Comment:
Now that I think of it, dummy data is better

Reply:
Yeah, will change